pkg/crypto/crypto.go

```diff
@@ -0,0 +1,104 @@
+package crypto
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/armor"
+)
+
+// EncryptUsername returns the encrypted value to be used as a username annotation
+func EncryptUsername(username string, key []byte) (string, error) {
```

Comment:
So re-reading and seeing it here, this looks like a generic function that can encrypt any string. Not sure if this should be called `EncryptUsername`. So maybe `Encrypt()` at which point the only thing this function does in addition to `encrypt()` is the funny business with the newline.

Reply:
This comment still holds. Basically `Encrypt()` function returns an encrypted value that can be used as a JSON string.

Reply:
This is true. Should I go ahead with a renaming of `EncryptUsername` & `DecryptUsername` to `EncryptAsJSONString`  & `DecrpytJSONString`?

Reply:
That would be `EncryptToJSONString` and `DecryptToJSONString`

Reply:
I will assume you mean `DecryptFromJSONString` :slightly_smiling_face: